app/src/main/java/com/story/mipsa/attendancetracker/Settings.java

```diff
@@ -0,0 +1,101 @@
+package com.story.mipsa.attendancetracker;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.ActionBar;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.content.ContextCompat;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.content.Intent;
+import android.graphics.Color;
+import android.graphics.drawable.ColorDrawable;
+import android.os.Bundle;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+
+import java.util.ArrayList;
+
+public class Settings extends AppCompatActivity {
+    private BottomNavigationView bottomNavigationView;
+    private ArrayList<String> settingNames = new ArrayList<>();
+    private ArrayList<Integer> settingImages = new ArrayList<>();
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        overridePendingTransition(R.anim.fade_in, R.anim.fade_out);
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_settings);
+
+        this.getSupportActionBar().setDisplayOptions(ActionBar.DISPLAY_SHOW_CUSTOM);
+        getSupportActionBar().setDisplayShowCustomEnabled(true);getSupportActionBar().setCustomView(R.layout.custom_action_bar);
```

Comment:
Move `getSupportActionBar().setCustomView(R.layout.custom_action_bar);` to a new line